CMakeLists.txt

```diff
@@ -3,14 +3,15 @@
 #
 cmake_minimum_required(VERSION 3.15)
 
+include(GNUInstallDirs)
```

Comment:
This should be under `project` declaration, otherwise cmake will throw a warning:
```
  Unable to determine default CMAKE_INSTALL_LIBDIR directory because no
  target architecture is known.  Please enable at least one language before
  including GNUInstallDirs.
Call Stack (most recent call first):
  CMakeLists.txt:6 (include)
```

Reply:
Done: https://github.com/giladreich/cmake-imgui/pull/7/commits/40a0c05ccbe572e5527756887388acf3b3895904